doc/classes/MeshInstance.xml

```diff
@@ -22,6 +22,7 @@
 			<return type="void">
 			</return>
 			<description>
+                This helper creates a [MeshInstance] child [Node] with gizmos at every vertex calculated from the mesh geometry. It's mainly used for testing.
```

Comment:
You should use tabs for indentation like on the other lines. Also, I would just use "node" and not "[Node]", the hyperlink is overkill here when the actual node type we're interested in is MeshInstance.

Could you fix that and squash the commits together? We need only one commit for the complete change.